### PR TITLE
Script for distaster recovery

### PR DESCRIPTION
**Pull request by slyfer**

### CODE REVIEW

File: bin/dump-postgres.sh
Comment (brushtyler) [inline bin/dump-postgres.sh]: The command acts on `pwd` while expecting to be executed on `DUMPDIR`.
Furthermore it's not clear how many entries it will keep (add a comment, please)

File: bin/dump-postgres.sh
Comment (brushtyler) [inline bin/dump-postgres.sh]: why do you run it again (see line 8) and having a different `tail` argument?

File: bin/dump-postgres.sh
Comment (brushtyler) [inline bin/dump-postgres.sh:4]: consider to create `dump` dir if missing to avoid failure.

File: bin/dump-postgres.sh
Comment (brushtyler) [inline bin/dump-postgres.sh]: if the command execution fails for any reason, `DESTFILE` is created anyway.
You should check return code and, if the command is failed, drop `DESTFILE` and report failure w/o changing `LATEST`.

**Review by brushtyler — COMMENTED** [review]
It would also be useful to log errors to file (e.g. using `logger -s <msg>`). It would help once the scripts are executed as cron jobs, but it can be done later (I will open a new issue for it).

File: bin/dump-postgres.sh
Comment (slyfer) [reply bin/dump-postgres.sh]: It's an error

File: bin/dump-postgres.sh
Comment (slyfer) [reply bin/dump-postgres.sh:4]: done

File: bin/dump-postgres.sh
Comment (slyfer) [reply bin/dump-postgres.sh]: I don't know how to do this, please help me

Comment (brushtyler) [reply bin/dump-postgres.sh]: something like:
```bash
docker exec noiapp_noiapp-database_1 bash -c 'pg_dump --create --format t -U noiapp' >$DESTFILE
RET=$?
if [ $RET -ne 0 ] ; then
  rm $DESTFILE
  echo "DB dump has failed, $DESTFILE not created." >&2 
  exit $RET
fi
```
not tested (I cannot do it by now) but you can give it a try by using a not-existing user as `pg_dump` argument

Comment (slyfer) [reply bin/dump-postgres.sh]: thanks, it works! (tested with no container and with wrong user)

File: bin/dump-postgres.sh
Comment (slyfer) [reply bin/dump-postgres.sh]: Please can you explain? I don't understand

**Review by brushtyler — APPROVED** [review]
@slyfer: well done.

**Review by brushtyler — APPROVED** [review]

**Comment by brushtyler** [comment]
@slyfer unfortunately I cannot merge it (no write access), so my approval is not enough.

**Pull request merged**
